src/content/guide/tools-and-features/system-firmware.md

```diff
@@ -0,0 +1,170 @@
+- **Hardware abstraction**: Providing a single, unified interface to the device, regardless of the underlying hardware architecture
+- **Application enablement**: Exposing a feature-rich API that is used by developers to write applications for the device
+- **Over-the-air updates**: Allowing rapid remote changes to code running on the device while providing resilience in poor connectivity environments, ensuring your device is never bricked
+
+Unlike application firmware, system firmware is written and maintained primarily by the Particle team. This is a conscious decision meant to keep you focused on your particular use case without needing to understand the nuances of low-level device behaviors.
+
+That being said, Particle's firmware repository is available as an open source project for those that want deep visibility into system firmware. To view the code and/or contribute, check out the [repo on GitHub](https://github.com/spark/firmware).
+
+
+## Versioning
+New features, security patches, and bugfixes are introduced to new versions of system firmware regularly by the Particle team. These changes to system firmware are bundled into _releases_ that are versioned using [semantic versioning](http://semver.org/) best practices.
+
+System firmware versions that are suffixed with `-rc.x` are called _prereleases_ ("rc" stands for release candidate). These prereleases contain the changes that will eventually become a default release, but still need more thorough usage and testing. We recommend that you **do not** flash prereleased firmware to your production units deployed in the field.
+
+Each release is documented thoroughly to give you a comprehensive picture of what has changed in the new version. For a full list of system firmware releases and their descriptions, please check out [the release page on GitHub](https://github.com/spark/firmware/releases) for the firmware repository.
+
+## Firmware Dependencies
+Recall from the introduction of this guide that system firmware provides a foundation for the application firmware that you write to run. As such, it is important to understand the tight coupling between application and system firmware.
+
+Application firmware that is written in the [Web](https://build.particle.io) or [Desktop](https://www.particle.io/products/development-tools/particle-desktop-ide) IDEs are _compiled against_ a specific version of system firmware before being sent to a device to run. That is, the system firmware acts as a translator - taking the human-readable code you write using the firmware API and translating into machine-executable code that the device can run.
+
+This creates a dependency that must be carefully managed. For example, imagine a new firmware primitive was introduced in system firmware version `1.0.0`, `Particle.travelInTime()`. As an aspiring time traveler, you quickly add the new feature to your firmware logic and send the code off to be compiled and flashed to your Electron.
```

Comment:
I'd complete the explanation of the rule before jumping into an example.

This creates a dependency that must be carefully managed. A user application can only run on a device with the same or newer system firmware version than the system firmware used to compile the user application. This is because the user application may be using new functionality that was not available in older system firmware. Allowing a new application to run with older system firmware might lead to a crash.